interactions/client/bot.py

```diff
@@ -509,7 +509,7 @@ def __resolve_commands(self) -> None:
             This is an internal method. Do not call it unless you know what you are doing!
         """
         for cmd in self._commands:
-            if cmd.resolved:
+            if cmd.coro.__qualname__ in [Cmd.__qualname__ for Cmd in self.__command_coroutines]:
```

Comment:
why are you doing this change? it feels unnessecary to do an extra list comprehension for... for what even?

Reply:
It changed commands from being resolved globally after loaded by one client to being resolved once per client

Reply:
Hm I see. still it would make more sense to have it before the loop wouldn't it?

Reply:
LMAO fair ok